src/colorize.cr

```diff
@@ -111,9 +111,15 @@ module Colorize
     self.enabled = STDOUT.tty? && STDERR.tty?
   end
 
-  def self.reset(io = STDOUT)
+  # Resets the color and mode of *io* if `enabled` is `true`.
+  def self.reset(io : IO = STDOUT)
     io << "\e[0m" if enabled?
```

Comment:
You can use `Colorize.reset(io) if enabled?`

Reply:
But this is the definition of `Colorize.reset(IO)`. Why should I use it inside of it? That would cause a stack overflow.